postgres-appliance/scripts/configure_spilo.py

```diff
@@ -188,6 +188,9 @@ def deep_update(a, b):
       {{#STANDBY_PORT}}
       port: {{STANDBY_PORT}}
       {{/STANDBY_PORT}}
+      {{#STANDBY_SLOT}}
+      primary_slot_name: {{STANDBY_SLOT}}
+      {{/STANDBY_SLOT}}
```

Comment:
I think this one is a bit more consistent with remaining env variable names:
```suggestion
      {{#STANDBY_PRIMARY_SLOT_NAME}}
      primary_slot_name: {{STANDBY_PRIMARY_SLOT_NAME}
      {{/STANDBY_PRIMARY_SLOT_NAME}}
```

Reply:
No, I am talking about Spilo docs: https://github.com/zalando/spilo/blob/master/ENVIRONMENT.rst

Reply:
I saw that the other `STANDBY_*` were not documented, I added some of them. (even though the best doc is configure_spilo.py ;) )

I'm stubborn, but do you really think patroni doc is ok?

https://patroni.readthedocs.io/en/latest/replica_bootstrap.html

> You can use Patroni’s permanent replication slots feature on the primary cluster to maintain a replication slot with the same name as primary_slot_name, or its default value if primary_slot_name is not provided.

https://patroni.readthedocs.io/en/latest/SETTINGS.html

> primary_slot_name: which slot on the remote master to use for replication. This parameter is optional, the default value is derived from the instance name (see function slot_name_from_member_name).

Reply:
Hi,

I didn't forget about this @CyberDem0n, I just need a yes or no: )